CAIPs/caip-207.md

```diff
@@ -0,0 +1,236 @@
+---
+caip: 207
+title: JSON-RPC Authority Negotiation
+author: Pedro Gomes (@pedrouid), Hassan Malik (@hmalik88)
+discussions-to: https://github.com/ChainAgnostic/CAIPs/pull/207
+status: Draft
+type: Standard
+created: 2023-02-02
+updated: 2023-02-02
+requires: [2, 10, 25, 171]
+---
+
+## Simple Summary
+
+CAIP-207 extends CAIP-25 to allow callers and respondents to negotiate RPC
+authorities and RPC semantics before authorizing RPC terms. 
+
+## Abstract
+
+CAIP-207 defines additional properties that CAIP-25 can be used
+iteratively/progressively to layer custom or local RPC semantics and/or routing
+onto a session. Since CAIP-25 respondents ignore unknown properties, respondents
+that conform to CAIP-25 but not to CAIP-207 should be carefully considered and
+accomodated by implementers. 
+
+## Motivation
+
+While some core methods and notifications are foundational to a namespace and
```

Comment:
Are we still using the term "namespace"?

Reply:
heh, the reason we moved to `scope objects` for CAIP-25 namespaces is to disambiguate them from folders in the /namespaces/  repo, which we ARE still using for the time being 🙈